### DISCUSSION TITLE
Can I disable the left column header?

### DISCUSSION THREAD

**Discussion by chairedevil** [nick-keller/react-datasheet-grid#68] [Q&A] [answered]
According to the topic title.
Can I disable the left column header?

![スクリーンショット 2021-10-19 131607](https://user-images.githubusercontent.com/22753035/137843022-986ffb00-3556-4db3-b875-57f623396c87.jpg)

**Comment by nick-keller** [comment, marked as answer]
Hello @chairedevil ,
You can use the [guttercolumn](https://react-datasheet-grid.netlify.app/docs/api-reference/props#guttercolumn) prop to customize the left column. Then use the [title](https://react-datasheet-grid.netlify.app/docs/api-reference/columns#title) key to remove / replace the default triangle.

```ts
<DataSheetGrid gutterColumn={{ title: <div /> }} />
```

**Reply by chairedevil** [reply]
Thank you so much.

I handle this in the following way.
```
      '& .dsg-row': {
        '& .dsg-cell-gutter': {
          display: 'none',
        },
      },
```
`gutterColumn={{ width: '0 0 0', title: <div /> }}`

Please tell me, If you have a better way.

**Reply by nick-keller** [reply]
You want to remove the gutter completely? Then yes, this is the only way for now, might change in the future!
